src/Group.cpp

```diff
@@ -41,11 +41,19 @@ Group::~Group()
 
 }
 
-Group::Group(const RString& sDir, const RString& sGroupDirName) {
-    RString sPackIniPath = sDir + sGroupDirName + "/" + INI_FILE;
-    m_sPath = sDir + sGroupDirName;
-    m_sGroupName = Basename(sGroupDirName);
-    m_sDisplayTitle = m_sGroupName;
+Group::Group(const RString& sDir, const RString& sGroupDirName, bool bFromProfile) {
+    RString sPackIniPath;
+    if (bFromProfile) {
+        sPackIniPath = sDir + "/" + INI_FILE;
+        m_sPath = sDir;
+        m_sGroupName = CUSTOM_SONG_PATH;
```

Comment:
Oh god don't reference a program level global. Introduce a function in RageUtil to fetch it. Eventually when we clean up the code this will need to change. Same below.

Reply:
oop updated